src/index.scss

```diff
@@ -154,13 +165,22 @@ body {
     font-size: 1.8rem;
   }
 
+  .modal-content {
+    overflow-y: scroll;
+  }
+
```

Comment:
Default behaviour is better in this case (`overflow: auto`).

_"If content is overflowing add a scrollbar"_